packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -24,9 +24,8 @@
     "Programming Languages"
   ],
   "dependencies": {
-    "@salesforce/aura-language-server": "2.3.6",
-    "@salesforce/lightning-lsp-common": "2.3.6",
-    "@salesforce/lwc-language-server": "2.3.6",
```

Comment:
Was this dep accidentally included in this package?

Reply:
No, the aura lang server used to have to directly depend on lwc lang server for some information but that dependency went away with the latest changes in lwc.

Reply:
Nevermind, I still have to sort out some dependencies at the lang server levels so I'll keep this around for now.